### DISCUSSION TITLE
When self-hosting, tables cannot be deleted properly in NHost Dashboard.

### DISCUSSION THREAD

**Discussion by S0z1e** [nhost/nhost#3113] [Q&A]
**Describe the bug**
 tables cannot be deleted properly.

**To Reproduce**
1. Go to 'DataBase'
2. Select todos table
3. Right Click on 'delete table'
4. popup say "Failed to fetch"

**Expected behavior**
should be deleted todos table

**Screenshots**
<img width="834" alt="image" src="https://github.com/user-attachments/assets/bf4eb496-bb73-4dd8-a96c-f5f6552cf496" />
<img width="1002" alt="image" src="https://github.com/user-attachments/assets/74229253-3865-435f-ae2c-87ab8470a42b" />

**Comment by dbarrosop** [comment]
Moving to discussions as this isn't a bug, there is virtually no difference between self-hosting and the CLI and this works fine in the CLI. My guess is you have something misconfigured. Make sure all your [env vars](https://github.com/nhost/nhost/tree/main/dashboard#environment-variables-for-local-development-and-self-hosting) are properly configured for your self-hosting environment.
